modules/katamari/src/test/ts/atomic/api/optional/OptionalGetOrTest.ts

```diff
@@ -1,7 +1,7 @@
 import { Assert, UnitTest } from '@ephox/bedrock-client';
+import fc from 'fast-check';
 import * as Fun from 'ephox/katamari/api/Fun';
```

Comment:
Is it weird that we have the fast-check import in-between two ephox imports?

Reply:
`ephox/...` is an alias import, so it's internal while fast-check is external.

Reply:
Another way of wording that, is this keeps it the same as if we used paths instead of the alias:
```
import fc from 'fast-check';
import * as Fun from '../../main/ts/ephox/katamari/api/Fun';
```